src/__experimental__/components/input-icon-toggle/input-icon-toggle.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
It might be a good idea to add some specs that are not simply snapshot testing, but rather tests the functionality in a more conventional, unit test manner.

Reply:
added a test to check if an `Icon` component is rendered without the `content` prop. Rest of it just tests the style changes based on props, so I think that snapshots are more proper for that.